sphinxcontrib/katex.py

```diff
@@ -362,7 +362,7 @@ class KaTeXServer:
     # wait for the server to stop in seconds
     STOP_TIMEOUT = 0.1
 
-    @staticmethod
+    @classmethod
     def timeout_error(self, timeout):
```

Comment:
Thanks for spotting this.

I would then also rename `self` to `cls`:
```suggestion
    def timeout_error(cls, timeout):
```